kolibri/utils/conf.py

```diff
@@ -25,6 +27,9 @@
 
 logger = logging.getLogger(__name__)
 
+with open(os.path.join(os.path.dirname(__file__), 'KOLIBRI_CORE_JS_NAME')) as f:
+    KOLIBRI_CORE_JS_NAME = f.read().strip()
```

Comment:
@benjaoming more 'single source of truth'ing here. Also, moved it into conf and out of settings, because this is not something an end user should be allowed to change (because this variable name has to be determined at time of the frontend build, so changing it at runtime would just break things).